wordpress-mobile/WordPress-iOS
# Implement fetch likes in PostService

Refs #15662 

Add implementation to fetch likes for a post in `PostService`. 

To test:

Confirm the app builds. The rest are covered through unit tests as this is a "backend" change.

PR submission checklist:
